cloudflare/resource_cloudflare_fallback_domain.go

```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+var default_domains = []string{
```

Comment:
we can do away with this now because i added support for this via cloudflare/cloudflare-go#784

Reply:
great.  any idea when that will be functional in the public api?  doesn't seem to work in my testing.

Reply:
public documentation is incoming however it already works. it is a DELETE request with the query parameters, and an empty JSON array just as the browser does.

anything that works via the UI has to work with the API as they are serviced from the same infrastructure.

Reply:
well, crap. yeah, i was using `&` instead of `?` to separate my query parameters in the curl test ... and the fact it requires a valid json body seemed odd too... but whatever. it does work.